stripper/ze_lotr_isengard_csgo1.cfg

```diff
@@ -362,4 +362,32 @@ modify:
 	{
 		"OnTrigger" "CooldownsRunScriptCodePushCooldown(90);0-1"
 	}
-}
\ No newline at end of file
+}
+
+;Renable radar (even if it doesn't have a minimap) so that players can see directions (north)
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "consola,Command,sv_disable_radar 0,4,-1"
```

Comment:
This is a shit fix, because it's going to be disabling/enabling radar at the start of every round, as well as send chat messages for the changes. Better to take it out of isengard.nut directly.

Reply:
After the last VScript fix I tried didn't work, I don't feel like trying make changes using custom VScript. As such, I have simply removed that section of the stripper and the radar will, although annoying, remain disabled.